### PR TITLE
Updated content for Individuals Income Received split

### PR DESCRIPTION
**Pull request by ben-HMRC**

### CODE REVIEW

File: source/documentation/additional-information.html.md [hunk 1]
```diff
@@ -6,50 +6,59 @@ description: Software developers, designers, product owners or business analysts
...
+If Making Tax Digital (MTD) customers are earning any type of income in addition to their self-employment and property income, they must report that to HMRC just as they would in their Self Assessment return. 
 
-The software should prompt customers to provide details about any additional income sources. If the software does not support submission of any applicable additional income, customers must use [HMRC online services](/guides/income-tax-mtd-end-to-end-service-guide/documentation/online-tax-account.html#hmrc-online-services) to make their submission. For more information about which type of income to tell HMRC, refer to [Check if you need to tell HMRC about additional income (GOV.UK)](https://www.gov.uk/check-additional-income-tax). 
+>Currently, the endpoints for reporting additional income through software are included in the Individuals Income Received API. As of June 2024, the Individuals Income Received API will be split into the following APIs:<ul><li>Individuals Employments Income API</li>
```
Comment (fmcgrath) [inline source/documentation/additional-information.html.md:11]: Why say 'June 2024'? The API has been split already and the 8 new APIs have already been deployed to PROD.

Also, if making a 'forward looking' statement, I suggest writing 'From June 2024,...'

Should we also say that the Individuals Income Received API will be deprecated/removed at some point?

File: source/documentation/additional-information.html.md [hunk 2]
```diff
@@ -339,17 +348,17 @@ After submitting the above losses, customers can view their loss position. If th
...
 
-Capital Gains Tax is a tax on the profit when someone sells or disposes of an asset that has increased in value. Customers may need to report on the following types of asset:
+Capital Gains Tax (CGT) is a tax on the profit when someone sells or disposes of an asset that has increased in value. Customers may need to report on the following types of asset:
 
 * residential property
 * other property 
 * listed and unlisted shares and securities 
 
-For more information on capital gains tax, see [Capital Gains Tax: what you pay it on, rates and allowances](https://www.gov.uk/capital-gains-tax).
+For more information on Capital Gains Tax, see [Capital Gains Tax: what you pay it on, rates and allowances](https://www.gov.uk/capital-gains-tax).
```
Comment (fmcgrath) [inline source/documentation/additional-information.html.md:357]: information about Capital Gains Tax

'information on' is associated with US English

File: source/documentation/additional-information.html.md (hunk 1 above)
Comment (ben-HMRC) [reply source/documentation/additional-information.html.md:11]: This was what was agreed with Nikhila - see https://jira.tools.tax.service.gov.uk/browse/MTDSA-23012

File: source/documentation/additional-information.html.md [hunk 3]
```diff
@@ -339,17 +348,17 @@ After submitting the above losses, customers can view their loss position. If th
...
 
-The customer can create, update, or delete their capital gains tax in software only after the end of the tax year. If they need to make changes during the year, they need to use HMRC online services. In some cases, customers must report and pay before they submit their final declaration. However, this information will still need to be included in their final declaration if they are a Self Assessment or MTD customer.
+The customer can create, update, or delete their CGT in software only after the end of the tax year. If they need to make changes during the year, they need to use HMRC online services. In some cases, customers must report and pay before they submit their final declaration. However, this information will still need to be included in their final declaration if they are a Self Assessment or MTD customer.
```
Comment (fmcgrath) [inline source/documentation/additional-information.html.md:362]: Can we write 'CGT' in normal text?

Although https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style and https://design.tax.service.gov.uk/hmrc-content-style-guide/ do not seem to outlaw 'CGT', abbreviations and acronyms are not encouraged and Capital Gains Tax is the preferred form.

**Review by fmcgrath — CHANGES_REQUESTED** [review]
A few minor wording issues. I presume that all the edited hyperlinks work - I don't have time to test them.

File: source/documentation/additional-information.html.md (hunk 2 above)
Comment (ben-HMRC) [reply source/documentation/additional-information.html.md:357]: I can't find anything to support that 'information on' is US English, seems like it is fine in British English. Also that is not part of the change that's being reviewed.

File: source/documentation/additional-information.html.md (hunk 1 above)
Comment (fmcgrath) [reply source/documentation/additional-information.html.md:11]: OK. It looks as if this issue has further to go. Will wave through for now.

**Review by fmcgrath — APPROVED** [review]
Waving through for now.

**Pull request merged**